FAQ.md

```diff
@@ -36,7 +36,7 @@ To prevent that exception from happening, add the following line before calling
 
 We have a few examples! The simplest of all is in [the root example folder](/example/). This folder must be called `example` and contain a single project, so it's a very simple generic example game.
 
-However there is an [examples folder inside the docs](/docs/examples) that contains many more examples. Most newer features we are adding we are trying to add a specific example for that. Those are directed to specific features.
+However there is an [examples folder inside the doc](/doc/examples) that contains many more examples. Most newer features we are adding we are trying to add a specific example for that. Those are directed to specific features.
```

Comment:
Just one tiny thing more, maybe rewrite this sentence a bit so that it fits with the singular?
Maybe:
"However there are many more [examples inside the doc folder](/doc/examples)."

Reply:
Haha, too slow. :)

Reply:
Sounds good, I already merged, but I will do that directly on the developer